### ISSUE TITLE
Cleanup devDependency into xXBJXx/iobroker-react.git#IssiVersion

### ISSUE THREAD

**Issue by mcm1957** [iobroker-community-adapters/ioBroker.tractive-gps-orig#2]
The following devDependency needs some cleanup:
```
    "iobroker-react": "git+https://git@github.com/xXBJXx/iobroker-react.git#IssiVersion",
```

**Comment by mcm1957** [comment]
siehe https://github.com/ioBroker/adapter-react-v5/issues/362

**Comment by mcm1957** [comment]
fixed with 2.0.0 by @GermanBluefox
